### DISCUSSION TITLE
Documentation of the ImportExcel Powershell module

### DISCUSSION THREAD

**Discussion by wyatt-wong** [dfinke/ImportExcel#1492] [General]
Is there documentation of the ImportExcel Powershell module which show the commands and the required parameters ?

**Comment by dfinke** [comment]
Not that I can think of. What required parameters issues are you running into?

```powershell
Get-Service | Export-Excel
```

![image](https://github.com/dfinke/ImportExcel/assets/67258/eb86fc6c-3ba9-4c53-8bf0-b7bc72004404)

**Reply by wyatt-wong** [reply]
I am not looking for a particular command but something which looks like this:

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/?view=powershell-7.3

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/add-content?view=powershell-7.3
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/clear-content?view=powershell-7.3
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/clear-item?view=powershell-7.3
...

etc

**Comment by dfinke** [comment]
No, sorry, nothing like that exists.

**Comment by wyatt-wong** [comment]
Actually I am asking to create some documentation for the ImportExcel PowerShell module. Of course I know it doesn't exists at all.

**Reply by dfinke** [reply]
Ah! Thank you for the offer.

James O'Neill (@jhoneill) started that effort.  

I believe I was the blocker.

In my mind this needs to be automated. The GitHub Action was not taken forward. I don't remember if I didn't take a PR forward. 

https://github.com/dfinke/ImportExcel/tree/master/mdHelp

May be time to revisit!
